Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -278,6 +277,97 @@ def is_tim_reviewer_needed(pr_files: list[str], support_label: str) -> bool:
     return False
 
 
+def get_user_from_ui_pr(pr) -> str:
+    """
+    Get user from PR that was opened from XSOAR UI by searching for the substring "Contribytor\n@" in the body of the PR
+    Arguments:
+    - pr - the opened PR
+
+    Returns:
+    - Found User
+    """
+    body = pr.body
+    index_of_user = body.find("Contributor\n@")
+    substring_user = body[index_of_user + len("Contributor\n@"):]
+    user_in_list = substring_user.split("\n")[0]
+    return user_in_list
+
+
+def find_all_open_prs_by_user(content_repo, pr_creator, pr_number) -> list:
+    """
+    find all open pr's that were opened by the same user as the current PR, excluding current PR
+    Arguments:
+    - content_repo: the content repository
+    - pr_creator: the author of the current PR
+    - pr_number: number of the current PR
+
+    Returns:
+    - list of all open PR's with similar author as the current PR
+    """
+    print(f'pr creator is: {pr_creator}')
+    all_prs = content_repo.get_pulls()
+    similar_prs = []
+    for pr in all_prs:
+        if pr.number == pr_number:  # Exclude current PR
+            continue
+        if pr.user.login == "xsoar-bot":
+            pr_creator_from_body = get_user_from_ui_pr(pr)
+            if pr_creator_from_body == pr_creator:
+                similar_prs.append(pr)
+        elif pr.user.login == pr_creator:
+            similar_prs.append(pr)
+        else:
+            continue
+    print(f'similar prs are: {similar_prs}')
+    return similar_prs
+
+
+def reviewer_of_prs_from_current_round(other_prs_by_same_user) -> str:
+    """
+    Get all PR's that are currently open from the same author, filter the list and return reviewer if reviewer is part
+    of the current contribution round
+    Arguments:
+    - other_prs_by_same_user - list of opened PR's
+
+    Returns:
+    - Reviewer of the found pr's
+    """
+    content_roles = load_json(CONTENT_ROLES_PATH)
+    content_reviewers, __, _ = get_content_reviewers(content_roles)
```

Comment:
you don't need to call this function, you already have content_reviewers pulled in the main